db/migrate/20210909142344_add_question_set_to_response_sets.rb

```diff
@@ -0,0 +1,12 @@
+class AddQuestionSetToResponseSets < ActiveRecord::Migration[6.1]
+  def change
+    add_reference :response_sets, :question_set, foreign_key: true, index: true
+    reversible do |dir|
+      dir.up do
```

Comment:
is there a way to write this so the down process is clearer? 

-

Reply:
Similar answer to above :)

Reply:
Same here, yes please to defining up and down methods.